lighthouse-core/audits/redirects.js

```diff
@@ -33,38 +33,48 @@ class Redirects extends Audit {
       .then(mainResource => {
         // redirects is only available when redirects happens
         const redirectRequests = Array.from(mainResource.redirects || []);
+
         // add main resource to redirectRequests so we can use it to calculate wastedMs
         redirectRequests.push(mainResource);
-        let totalWastedMs = 0;
 
+        let totalWastedMs = 0;
         const pageRedirects = [];
-        for (let i = 1; i < redirectRequests.length; i++) {
-          const request = redirectRequests[i - 1];
-          const nextRequest = redirectRequests[i];
-          const wastedMs = (nextRequest.startTime - request.startTime) * 1000;
 
-          // We skip the first redirect in our calculations but show it in the table below.
-          // We allow 1 redirect (www. => m.)
-          if (i > 1) {
-            totalWastedMs += wastedMs;
+        for (let i = 0; i < redirectRequests.length; i++) {
+          const initialRequest = redirectRequests[i - 1];
+          const redirectedRequest = redirectRequests[i];
+
+          // Only create a table (with the initial request) if there are > 1 redirects
+          if (!initialRequest) {
+            if (redirectRequests.length > 1) {
```

Comment:
This `>1` and a `=== 2` below could instead use a `ACCEPTED_REDIRECTS` const if folks prefer.